static/js/documentation.js

```diff
@@ -88,3 +91,27 @@ $(function() {
     }, 19 * 1000); // Let first loop run through 19 seconds
   }
 });
+
+/**
+ * Sends a GET request to the specified URL with the given parameters.
+ * @param {string} url - The URL to request from.
+ * @param {object} headers - The headers to use in the GET request.
+ * @param {bool} [cache = false] - Whether or not to cache the returned results.
+ * @param {function(response)} callback - The function called when the AJAX request has completed successfully.
+ * @param {function} onFail  - The function called when the AJAX request fails.
+ */
+function fetchData(url, headers, cache = false, callback, onFail) {
+	$.ajax({
```

Comment:
looks like some space/tab issues throughout

Reply:
Switched from tabs to spaces, should look better now: https://github.com/cerner/fhir.cerner.com/pull/416/commits/a3d742237b990b3946cd5b537ca1713df3c04450 https://github.com/cerner/fhir.cerner.com/pull/416/commits/9b58fa354377ba88e5258666ed5707074632dcbf